invesalius/gui/widgets/canvas_renderer.py

```diff
@@ -448,13 +527,22 @@ def calc_text_size(self, text, font=None):
 
         w = 0
         h = 0
-        for t in text.split('\n'):
+        for t in text.split("\n"):
             _w, _h = gc.GetTextExtent(t)
             w = max(w, _w)
             h += _h
         return w, h
 
-    def draw_line(self, pos0, pos1, arrow_start=False, arrow_end=False, colour=(255, 0, 0, 128), width=2, style=wx.SOLID):
+    def draw_line(
+        self,
+        pos0,
+        pos1,
+        arrow_start=False,
+        arrow_end=False,
+        colour=(255, 0, 0, 128),
```

Comment:
Tuple of of 4 integers

Reply:
I annotated it as `Tuple[float, float, float, float]` because I saw the line
```python
        pen = wx.Pen(wx.Colour(*[int(c) for c in colour]), width, wx.SOLID)
```

Reply:
I think it's ok to be `float`. See this https://peps.python.org/pep-0484/#the-numeric-tower . This is happening in invesalius because vtk uses float colors (0.0 - 1.0) and is common to convert to int colors (0 - 255).